.github/workflows/awie.yml

```diff
@@ -42,8 +56,8 @@ jobs:
       veracode_srcclr_token: ${{ secrets.VERACODE_SRCCLR_TOKEN }}
 
   package:
-    needs: [get-version]
-    if: ${{ needs.get-version.outputs.stability != 'stable' }}
+    needs: [get-version, check-version]
+    if: ${{ needs.get-version.outputs.stability != 'stable' && !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}
```

Comment:
did you try `&& success()` or `&& !failure()` instead of `&& !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}`

Reply:
this may not work as intended since "different from cancelled" also includes "skipped" while success() may not